doc/examples/restore_dti.py

```diff
@@ -5,7 +5,7 @@
 
 The diffusion tensor model takes into account certain kinds of noise (thermal),
 but not other kinds, such as "physiological" noise. For example, if a subject
-moves during the acquisition of one of the diffusion-weighted samples, this
+moves during acquisition of one of the diffusion-weighted samples, this
```

Comment:
I'd dare to say that the article is necessary/correct.

Reply:
Let the maintainers review it, If it's wrong then I will change it.

Reply:
Both versions are ok. When both cases are acceptable, we normally prefer not changing it, so that the review is quicker for the maintainers.

Reply:
+1 with @jchoude comment. Both are ok.